lib/knexFunctions.js

```diff
@@ -14,19 +15,133 @@ function createGame(id){
   })
   .returning("id")
   .then(function(gameid){
-    allGames[gameid] = (gops.createGameObject("gops", id));
+    return knex('user_games')
+    .insert({
+      user_id: userid,
+      game_id: gameid[0]
+    }).returning("game_id")
+  }).then(function(gid){
```

Comment:
gid is never seen by anything, its just a placeholder for the value I'm getting. I was afraid of naming it gameid since I already use that name in the function. I'll rename it to newGameID though.